module/os/windows/spl/spl-lookasidelist.c

```diff
@@ -87,7 +87,7 @@ lookasidelist_cache_create(char *name,	/* descriptive name for this cache */
 		pLookasidelist_cache->cache_chunksize = size;
 
 		if (name != NULL) {
-			strncpy(pLookasidelist_cache->cache_name, name,
+			strlcpy(pLookasidelist_cache->cache_name, name,
```

Comment:
Just assumed this is copying around already null terminated strings
so others no problem just re null terminating them if they are one less than the buffer size.

Reply:
`strncpy()` will stop at end-of-string, and "forget" to null-byte it. `strlcpy` will always null-byte it, so it actually copies 1 byte less than what `strncpy()` used to do.